pkg/addons/adminconsole/customize.go

```diff
@@ -170,6 +178,28 @@ func (a *AdminConsoleCustomization) apply(ctx context.Context) error {
 	return nil
 }
 
+// clusterConfig makes sure a config map named "embedded-cluster-config" exists in the cluster.
+// XXX this will eventually be customizable by the user through the app release.
+func (a *AdminConsoleCustomization) clusterConfig(ctx context.Context) error {
+	kubeclient, err := a.kubeClient()
+	if err != nil {
+		return fmt.Errorf("unable to create kubernetes client: %w", err)
+	}
+	nsn := client.ObjectKey{Namespace: "kube-system", Name: "embedded-cluster-config"}
+	var cm corev1.ConfigMap
+	if err := kubeclient.Get(ctx, nsn, &cm); err == nil {
+		return nil
+	}
+	cm = corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Namespace: nsn.Namespace, Name: nsn.Name},
+		Data:       map[string]string{},
```

Comment:
could we toss the helmvm version in this to begin with?

Reply:
done.